### PR TITLE
PTFE Monitoring Guide v1

### PR DESCRIPTION
**Pull request by maryelizbeth**
[Asana](https://app.asana.com/0/331272348117180/778324875213431/f) 

This is oft-requested from our customers. At first pass, we'll offer a bare bones monitoring recommendation. 

In the future, we'll flesh out the guide a bit more -->  See: [Future Plans](https://app.asana.com/0/331272348117180/640938308249564/f)

### CODE REVIEW

File: content/source/docs/enterprise/private/monitoring.html.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+layout: "enterprise2"
+page_title: "Monitoring Private Terraform Enterprise"
+sidebar_current: "docs-enterprise2-private-installer-monitoring"
+---
+
+# Monitoring Terraform Enterprise Installer Migration
```
Comment (armchairlinguist) [inline content/source/docs/enterprise/private/monitoring.html.md]: To follow convention, this title would be something like "Monitoring a Private Terraform Enterprise Installer-based Instance" (or just Instance, if it's not installer specific).

File: content/source/docs/enterprise/private/monitoring.html.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+This document outlines best practices for monitoring a Private Terraform Enterprise (PTFE) instance.
+
+## Health Check 
+
+PTFE provides a `/_health_check` endpoint on the instance. It is a continuously polling endpoint that delivers as it promises: a health check. Polling this endpoint will display a real time report of the overall status of PTFE. If PTFE is up, the health check will return a `200 OK`.
```
Comment (armchairlinguist) [inline content/source/docs/enterprise/private/monitoring.html.md]: I think both of the middle sentences here aren't really needed.

File: content/source/docs/enterprise/private/monitoring.html.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+## Metrics & Telemetry
+
+Additional to health check monitoring we recommend monitoring PTFE as one would any other server. As a start we recommend monitoring and alerting on the following:
```
Comment (armchairlinguist) [inline content/source/docs/enterprise/private/monitoring.html.md]: In addition to health-check monitoring, we recommend monitoring standard server metrics on the PTFE instance:

File: content/source/docs/enterprise/private/monitoring.html.md (hunk 1 above)
Comment (maryelizbeth) [reply content/source/docs/enterprise/private/monitoring.html.md]: I think this was a bad copy-paste :face-palm: -- i'll fix that ASAP

File: content/source/docs/enterprise/private/monitoring.html.md (hunk 2 above)
Comment (maryelizbeth) [reply content/source/docs/enterprise/private/monitoring.html.md]: 👍

File: content/source/docs/enterprise/private/monitoring.html.md (hunk 3 above)
Comment (maryelizbeth) [reply content/source/docs/enterprise/private/monitoring.html.md]: ✅

**Pull request merged**